#制御周波数
hz: 10

#初期化時の値
#パーティクル数
num: 300
#初めにばらまくパーティクルの位置姿勢
init_x: 0.0
init_y: 0.0
init_yaw: 0.0
#ノイズを乗せるばらつき
x_cov: 0.5
y_cov: 0.5
yaw_cov: 0.5
distance_noise_ratio: 0.10
rotation_noise_ratio: 0.10
laser_noise_ratio: 0.040 #(m/m)
